### ISSUE TITLE
An option to encrypt the system by using only the unallocated space

### ISSUE THREAD

**Issue by lyrebird-sound-cloning-system** [calamares/calamares#1705]
**Is your feature request related to a problem? Please describe.**
Currently, disk encryption is tied to the erase disk option. If you have a backup data partition, another OS, it gets deleted. If I'm not mistake, Anaconda (Fedora installer) allows this.

**Describe the solution you'd like**
A way to encrypt the system and setting up necessary partitions using the unallocated space in the disk, without affecting other partitions.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by kkofler** [comment]
FYI, this is possible with manual partitioning.

**Comment by lyrebird-sound-cloning-system** [comment]
Yeah, but it seems that is [not currently possible](https://github.com/calamares/calamares/issues/1311#issuecomment-834171225) to create this scheme with LVM manually. And for some reason, the system doesn't boot if [manual partition with LUKS is done](https://github.com/calamares/calamares/issues/1311#issuecomment-846358911).

**Comment by lyrebird-sound-cloning-system** [comment]
This would be great for inexperienced users too and for ease installing when other systems such as Windows are present.
